aqua/smooth/Portfile

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        enzo1982 smooth 0.9.6 v
+github.tarball_from releases
+
+categories          aqua
+platforms           darwin
+license             Artistic-2
+maintainers         {@janosch mailbox.org:janosch1} openmaintainer
+description         ${name} is a class library for user interfaces written in C++
+long_description    ${name} is an object oriented C++ class library for Windows, macOS, Linux and most Unix-like operating systems. \
+                    It provides basic functionality and platform support for applications and libraries. \
+                    \n \
+                    \nFeatures provided by smooth include:\
+                    \n* user interface API with various widgets\
+                    \n* simple to use multithreading API\
+                    \n* file and network IO interface\
+                    \n* completely transparent Unicode and software internationalization support\
+                    \n* a libxml2 based XML parser\
+homepage            http://www.smooth-project.org/
+
+depends_lib         port:git \
+                    port:bzip2 \
+                    port:curl \
+                    port:fribidi \
+                    port:gtk3 \
+                    port:jpeg \
+                    port:libxml2
+
+use_configure       no
+
+build.args          prefix=${prefix}
+destroot.args       prefix=${prefix}
+
+patchfiles          patch-fix-libxml2-includes.diff
+
+set major           [lindex [split ${version} .] 0]
+set minor           [lindex [split ${version} .] 1]
+
+# The compatibility version of this library. This is defined in the Makefile in the REVISION variable.
+# The library naming scheme for smooth is major.minor.compat (NOT patch!)
+set compat 0
+
+# After building smooth, references to libsmooth and the id of libsmooth
+# are not pointing to ${prefix}/lib. We are fixing this here.
+post-destroot       {
+                      system "install_name_tool -change libsmooth-${major}.${minor}.${compat}.dylib \
```

Comment:
I'm surprised this works. You should use an absolute path to install_name_tool.

Reply:
Hmm, I dont understand. The usage of install_name_tool is 
`install_name_tool [-change old new] input`
I am using absoluth paths to reference the input and to specify the new install name, but for `old` I just have to use the current install name. If I specify something else here install_name_tool will fail because it does not know what to change.

Reply:
Done.